tools/protodoc/protodoc.py

```diff
@@ -498,9 +498,14 @@ def FormatFieldAsDefinitionListItem(outer_type_context, type_context, field, pro
         manifest_description.edge_config)
   else:
     formatted_security_options = ''
-
-  comment = '(%s) ' % ', '.join([FormatFieldType(type_context, field)] +
-                                field_annotations) + formatted_leading_comment
+  pretty_label_names = {
+      field.LABEL_OPTIONAL: '',
+      field.LABEL_REQUIRED: 'required ',
+      field.LABEL_REPEATED: 'repeated ',
```

Comment:
Yep, maybe add bold or italics or something to make it clear that this is distinguished.

Reply:
Added bold font.